File: Shared/Blog.Contracts/V1/Responses/Chart/ChartDataModel.cs
using System.Collections.Generic;

namespace Blog.Contracts.V1.Responses.Chart;

/// <summary>
/// The Chart Data model.
/// </summary>
public class ChartDataModel
{
    /// <summary>
    /// Gets or sets name.
    /// </summary>
    public string Name { get; set; }

    /// <summary>
    /// Gets or sets series.
    /// </summary>
    public List<ChartItem> Series { get; set; } = [];
}

File: Shared/Blog.FSharp.Contracts/V1/Responses/Chart/ChartDataModel.fs
namespace Blog.FSharp.Contracts.V1.Responses.Chart

open System.Collections.Generic

/// <summary>
/// Chart data model.
/// </summary>
type ChartDataModel() =
    /// <summary>
    /// Gets or sets the name.
    /// </summary>
    member val Name: string = "" with get, set

    /// <summary>
    /// Gets or sets the series.
/// </summary>
    member val Series: List<ChartItem> = List<ChartItem>() with get, set

